server/middleware/request.js

```diff
@@ -1,8 +1,21 @@
 const { v4: uuid } = require('uuid');
 const logger = require('../libs/logger');
-const { ValidationError } = require('../models/error');
+const { ValidationError, GenericError } = require('../models/error');
 const { isProduction } = require('../config');
 const listService = require('../services/list/');
+const { AxiosError } = require('axios');
+
+function axiosErrorMiddleware(err, _req, _res, next) {
```

Comment:
We could add tests for this in request.spec.js

Reply:
This has been completed 🙂